documentation/modules/configuring/con-config-kafka-bridge.adoc

```diff
@@ -17,60 +17,75 @@ For a deeper understanding of the Kafka Bridge cluster configuration options, re
 .Example `KafkaBridge` custom resource configuration
 [source,yaml,subs="+quotes,attributes"]
 ----
+# Basic configuration (essential)
 apiVersion: {KafkaBridgeApiVersion}
 kind: KafkaBridge
 metadata:
   name: my-bridge
 spec:
+  # Replicas (essential)
   replicas: 3 # <1>
+  # Kafka bootstrap servers (essential)
   bootstrapServers: _<cluster_name>_-cluster-kafka-bootstrap:9092 # <2>
+  # TLS configuration (optional)
   tls: # <3>
     trustedCertificates:
       - secretName: my-cluster-cluster-cert
         pattern: "*.crt"
       - secretName: my-cluster-cluster-cert
         certificate: ca2.crt
+  # Authentication (recommended)
```

Comment:
I would say these should be both optional or both recommended. As it here depends on how the Kafka cluster is configured and it is not really a choice here, I would maybe use optional for both?